packages/playground/src/components/dynamic_tabs.vue

```diff
@@ -49,6 +50,7 @@ export interface Tab {
   value: string;
   icon?: string;
   imgPath?: string;
+  workers?: number;
```

Comment:
can we make the name of the property more generic to refer to number of disks later if we need to ?
what do you think ?

Reply:
Maybe we can add another prop for the disks as well if needed later on.